### PR TITLE
Bump junit dependencies

### PR DESCRIPTION
**Pull request by SuperQ**
Update junit version requirement to fix CVE-2020-15250 warnings.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: simpleclient_hibernate/pom.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>[4.13.1,)</version>
```
Comment (brian-brazil) [inline simpleclient_hibernate/pom.xml:56]: Please stick with a specific version, and keep it the same in all the poms.

Comment (SuperQ) [reply simpleclient_hibernate/pom.xml]: Done. It looks like these two were the only ones left behind.

Comment (brian-brazil) [reply simpleclient_hibernate/pom.xml]: Hmm, what about spring_web?

**Comment by brian-brazil** [comment]
spring_web still needs updating.

**Comment by SuperQ** [comment]
Ping @fstab @tomwilkie

**Review by roidelapluie — APPROVED** [review]

**Pull request merged**